roles/nuage_master/tasks/serviceaccount.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Copy Configuration to temporary conf
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{nuage_tmp_conf}}
```

Comment:
Instead of setting nuage_tmp_conf in vars/main.yml, I would suggest using mktemp to create the directory for the config:
```
- name: Create temporary directory for admin kubeconfig
  command: mktemp -u /tmp/openshift-ansible-XXXXXXX.kubeconfig
  register: nuage_tmp_conf_mktemp
  changed_when: False

- set_fact:
    nuage_tmp_conf: "{{ nuage_tmp_conf_mktemp.stdout }}"
```